### ISSUE TITLE
Need to add notes section.

### ISSUE THREAD

**Issue by ritiku2004** [ClubToCode/Notzy#7]
Currently, our to-do app lacks a notes feature, which limits users' ability to add extra details or context to their tasks. Adding a notes functionality will enhance user experience and provide a more comprehensive task management solution.

1.Integrate a notes input field.
2.Enable users to add, edit, and delete notes.

**Comment by Ronit-gurjar** [comment]
okay, would you like to work on this issue?

**Comment by ritiku2004** [comment]
Yes, I want to solve this problem. and I can add edit and more buttons to To-do section.

**Comment by Ronit-gurjar** [comment]
you can work on this issue. for the todo-edit button please create a separate issue

**Comment by ritiku2004** [comment]
okay.
